### DISCUSSION TITLE
How-To: Get Access Token by Login Token.

### DISCUSSION THREAD

**Discussion by maximilianosinski** [JohnnyCrazy/SpotifyAPI-NET#768] [Q&A]
When creating a Account with the Spotify API, it gives back a login_token, How do i make a Access Token out of it.

**Comment by JohnnyCrazy** [comment]
On which page do you get a login token? The only way to get an access token is via specific OAuth2 flows

https://developer.spotify.com/documentation/general/guides/authorization/
